src/state/store.js

```diff
@@ -0,0 +1,43 @@
+import { combineReducers, createStore, applyMiddleware } from 'redux';
+// import { persistReducer } from 'redux-persist';
+// import storage from 'redux-persist/lib/storage'; // defaults to localStorage for web
+import thunk from 'redux-thunk';
+
+import {
+  userReducer,
+  courseReducer,
+  moduleReducer,
+  programReducer,
+} from './ducks';
+
+const rootReducer = combineReducers({
+  programReducer,
+  userReducer,
+  courseReducer,
+  moduleReducer,
+});
+
+// const persistConfig = {
+//   key: 'coursesList',
+//   storage,
+// };
+
+// redux-persist used to keep state from wiping after refreshes
+// const persistedCourseReducer = persistReducer(persistConfig, rootReducer);
+
+const middleware = [thunk];
+const createStoreWithMiddleware = applyMiddleware(...middleware)(createStore);
+
+// export const store = createStoreWithMiddleware(
```

Comment:
temporarily commented out persisted store b/c it was feeling really sticky for me. not sure if that was my fault or what but decided to go back to the more primitive approach for the night